src/components/ConfirmationBox/ConfirmationBox.test.js

```diff
@@ -21,22 +21,22 @@ it('calls setActualItem when yes is clicked', () => {
 	const mockProps = getProps();
 
 	const wrapper = shallow(<ConfirmationBox {...mockProps} />);
-	wrapper.find({testAttribute: 'YES'}).simulate('click');
+	wrapper.find({testattribute: 'YES'}).simulate('click');
```

Comment:
I'll create a separate task for these to be consistent (e.g. data-test-id or selector)